src/Data/Geometry/Structure/Feature.hs

```diff
@@ -99,6 +101,34 @@ sjLat (ScenarioJSON js) = getProp "lat" js
 sjAlt :: ScenarioJSON -> Maybe Float
 sjAlt (ScenarioJSON js) = getProp "alt" js
 
+sjBlockColor :: ScenarioJSON -> Maybe (JSString)
+sjBlockColor sj = getHexColor "defaultBlockColor" sj
+sjActiveColor :: ScenarioJSON -> Maybe (JSString)
+sjActiveColor sj = getHexColor "defaultActiveColor" sj
+sjStaticColor :: ScenarioJSON -> Maybe (JSString)
+sjStaticColor sj = getHexColor "defaultStaticColor" sj
+sjLineColor :: ScenarioJSON -> Maybe (JSString)
```

Comment:
Minor cosmetic thing: remove unnecessary parentheses.
In general, you can try to use `hlint` program on the source code. It can spot such things and suggest style improvements. Though it breaks sometimes on files with foreign import javascript.
You can install hlint via stack (stack install hlint).